web/src/containers/ScheduleSummary.js

```diff
@@ -16,21 +16,68 @@ const ScheduleSummary = ({ activities }) => (
             {t('scheduleSummary.noDataMessage')}
           </div>
         ) : (
-          activities.map(({ name: activityName, dateRange, milestones }, i) => (
+          <table className="budget-table">
+            <caption className="ds-u-visibility--screen-reader">
+              Activity List Overivew
+            </caption>
+            <thead>
+              <tr className="budget-table--row__primary-header">
+                <th scope="col">Activity List Overview</th>
+                <th scope="col" className="ds-u-text-align--left">
+                  Activity Date Range
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {activities.map(({ name: activityName, dateRange }, i) => (
+                <tr
+                  key={activityName}
+                  className="summaery-table--gray_row__highlight"
+                >
+                  <td
+                    className="ds-u-font-weight--bold ds-u-border-right--0"
+                    style={{ width: '70%' }}
+                  >
+                    Activity {i + 1}: {activityName} Milestones
+                  </td>
+                  <td className="ds-u-font-weight--bold ds-u-padding-right--3 ds-u-text-align--left ds-u-border-left--0 budget-table--cell__nowrap">
+                    {dateRange}
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        )}
+      </Subsection>
+      <Subsection
+        id="schedule-summary-table"
+        resource="scheduleSummary.milestone"
+      >
+        {activities.length === 0 ? (
+          <div className="ds-c-alert ds-c-alert--warn">
+            {t('scheduleSummary.noDataMessage')}
+          </div>
+        ) : (
+          activities.map(({ name: activityName, milestones }, i) => (
             <table key={activityName} className="budget-table">
               <caption className="ds-u-visibility--screen-reader">
                 Activity {i + 1}: {activityName}
               </caption>
               <thead>
+                <tr className="budget-table--row__primary-header">
+                  <th scope="col" style={{ width: '70%' }}>
```

Comment:
Is the goal to push the date column as far to the right as possible? If so, would it be better to set the width of the name column to 100% and let the browser do the math to figure out how wide it can really be?

Reply:
@mgwalker I tried taking off the 70% and the right column is all over the place -- based on the width of the left column text is where the right column text started.  So the 70% sets the starting point of the right column.  70% is the point at which it looks "nice".   You're right about 100% -- I tried it at 100% and the right column was pressed all the way over to the right.  It looked a little squished so I moved it back to 70%.

Reply:
I forgot to mention...  70% is what it had been before.  I didn't add it, just moved it up from where it had been used before.